### PR TITLE
Metadata update | Bulk Update

### PR DESCRIPTION
**Pull request by GitHubber17**
Remove `ms.prod`
Update `ms.service`

### CODE REVIEW

**Comment by MeeraDi** [comment]
@kexugit
@danieljurek

This PR is ready for review and merge.
Thanks!

**Comment by v-emilypr** [comment]
#sign-off

File: api/overview/azure/cognitiveservices/face-readme.md [hunk 1]
```diff
@@ -4,10 +4,10 @@ author: PatrickFarley
 ms.author: pafarley
 ms.date: 06/21/2021
 ms.topic: reference
-ms.prod: azure
 ms.technology: azure
 ms.devlang: csharp
-ms.service: face-api
+ms.service: cognitive-services
```
Comment (danieljurek) [inline api/overview/azure/cognitiveservices/face-readme.md:9]: This should be updated in https://github.com/Azure/azure-sdk/blob/main/_data/releases/latest/dotnet-packages.csv

File: api/overview/azure/latest/communication.networktraversal-readme.md [hunk 2]
```diff
@@ -5,10 +5,9 @@ author: acsdevx-msft
 ms.author: acsdevx-msft
 ms.date: 02/09/2022
 ms.topic: reference
-ms.prod: azure
 ms.technology: azure
 ms.devlang: dotnet
-ms.service: communication
+ms.service: azure-communication-services
```
Comment (danieljurek) [inline api/overview/azure/latest/communication.networktraversal-readme.md:10]: This should be updated in https://github.com/Azure/azure-sdk/blob/main/_data/releases/latest/dotnet-packages.csv

File: api/overview/azure/latest/extensions.aspnetcore.configuration.secrets-readme.md [hunk 3]
```diff
@@ -5,10 +5,9 @@ author: jsquire
 ms.author: jsquire
 ms.date: 04/06/2022
 ms.topic: reference
-ms.prod: azure
 ms.technology: azure
 ms.devlang: dotnet
-ms.service: extensions
+ms.service: key-vault
```
Comment (danieljurek) [inline api/overview/azure/latest/extensions.aspnetcore.configuration.secrets-readme.md:10]: This should be updated in https://github.com/Azure/azure-sdk/blob/main/_data/releases/latest/dotnet-packages.csv

File: api/overview/azure/latest/microsoft.azure.servicebus-readme.md [hunk 4]
```diff
@@ -5,10 +5,9 @@ author: ramya-rao-a
 ms.author: ramyar
 ms.date: 11/08/2021
 ms.topic: reference
-ms.prod: azure
 ms.technology: azure
 ms.devlang: dotnet
-ms.service: servicebus
+ms.service: service-bus
```
Comment (danieljurek) [inline api/overview/azure/latest/microsoft.azure.servicebus-readme.md:10]: This should be updated in https://github.com/Azure/azure-sdk/blob/main/_data/releases/latest/dotnet-packages.csv

**Review by danieljurek — COMMENTED** [review]
In `-readme.md` files the services need to be updated in the CSV file: https://github.com/Azure/azure-sdk/blob/main/_data/releases/latest/dotnet-packages.csv

I pointed out a few examples in this PR but there are a lot of changes here which need corresponding changes in that CSV file or on the next release the `ms.service` value will be overwritten. by a vaule that we generate.

**Comment by GitHubber17** [comment]
Closing per @MeeraDi

**Pull request closed without merging**